RELEASE-NOTES.md

```diff
@@ -104,24 +104,97 @@ buckets.default.allow_mult = true
 
 Note: this is defined in riak_kv.schema
 
-#### Backend configuration
+#### `riak_kv` configuration
 
-For Riak 2.0.5 or older, Old style backend configuration same as 1.5
-should be written in `riak_kv` section of `advanced.config`. See
+There are two style to configure Riak 2.0 behind Riak CS:
+
+1. Reuse `app.config` of Riak 1.4
+2. Copy proper items in old `app.config` to `advanced.config`
+
+In case of 1., `add_paths` should be changed to new Riak CS binaries
+installed by Riak CS 2.0 package, from
+`"/usr/lib/riak-cs/lib/riak_cs-1.5.4/ebin"` to
+`"/usr/lib/riak-cs/lib/riak_cs-2.0.0/ebin"`.  ```
+
+If Riak-2.0-native way of configuration is preferred, 2. is the
+choice.  Copy all `riak_kv` configuration items of `app.config` to
+`advanced.config` as it is, with `add_paths` updated.
+
+This is because old style backend configuration same as 1.5 should be
+written in `riak_kv` section of `advanced.config`. See
 [Setting up the Proper Riak Backend](http://docs.basho.com/riakcs/1.5.4/cookbooks/configuration/Configuring-Riak/#Setting-up-the-Proper-Riak-Backend)
-for details.
+for details. `app.config` should be removed when `advanced.config`
+should be used.
+
+#### Redesign of memory sizing
+
+It is also worth noting that since LevelDB has changed its
+configuration item on memory size configuration, from `max_open_files`
+to `total_leveldb_mem_percent`. This specifies total amount of memory
+consumed by LevelDB. This configuration is very important because the
+default value of memory cap limitation has changed from proportional
+number of `max_open_files` to specifying percentage of system's
+physical memory size.
+
+Configuring `total_leveldb_mem_percent` is *strongly recommended* as
+its default value is
+[70%](https://github.com/basho/eleveldb/blob/2.0.2/priv/eleveldb.schema#L17),
+which might be too aggressive for multi-backend configuration that
+also uses bitcask. Bitcask keeps its keydir on memory, which could be
+fairly large depending on use cases.
+
+Hereby basic idea of redisigning memory sizing is described: estimate
+total number of objects with their size distribution. Total number of
+keys `N(b)` in Bitcask accross the whole cluster will be:
+
+```
+N(b) = N(o, size <= 1MB) + avg(o, size > 1MB)
```

Comment:
In right hand side, it seems that the first term is extensive but second is intensive.